server/game/cards/06-WoE/JarlSvend.js

```diff
@@ -0,0 +1,22 @@
+const Card = require('../../Card.js');
+
+class JarlSvend extends Card {
+    // After an enemy creature is destroyed fighting Jarl Svend, make
+    // a token creature. (Jarl Svend must survive the fight.)
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardDestroyed: (event, context) =>
```

Comment:
I need to test tokens with cards that have onDestroyedEvents.

Reply:
I just added a test for Jarl destroying an enemy token and it seems to work as expected.